include/nighthawk/client/output_formatter.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "envoy/common/pure.h"
+
+#include "api/client/output.pb.validate.h"
+
+namespace Nighthawk {
+namespace Client {
+
+/**
+ * Transforms from nighthawk::client::Output to an output format.
+ */
+class OutputFormatter {
+public:
+  virtual ~OutputFormatter() = default;
+  virtual void setProto(const nighthawk::client::Output& output) PURE;
```

Comment:
Why not just have a `std::string formatProto(proto)` method?

Reply:
good point, done.